src/radical/pilot/agent/lm/fork.py

```diff
@@ -26,7 +26,7 @@ def _configure(self):
     # --------------------------------------------------------------------------
     #
     @classmethod
-    def lrms_config_hook(cls, name, cfg, lrms, logger):
+    def lrms_config_hook(cls, name, cfg, lrms, logger, profile):
```

Comment:
Here also it is `profile`. I think it is better if it is profiler

Reply:
fixed in 64ba387d - thanks.